index.qmd

```diff
@@ -157,6 +157,112 @@ Blurb on what Support Vector Machines are.
 
 ## Results
 
+```{r}
+n_obs <- nrow(df)
+```
+
+
+```{r}
+#| label: patient_demo
+#| eval: true
+#| echo: false
+#| warning: false
+#| tbl-caption: ""
+patient_demo <- df |>
+  dplyr::ungroup() |>
+  dplyr::select(c("age_at_scan", "gender", "ethnicity")) |>
+    gtsummary::tbl_summary() |>
+  gtsummary::modify_caption("Demographics of study population")
+patient_demo
+print(colnames(patient_demo))
+```
+
+@patient_demo shows the demographics of patients included in this study. A total of `r n_obs` patients were included in
+this study with a median (IQR) age of  `r gtsummary::inline_text(patient_demo, variable="age_at_scan")`.
+
+```{r}
+#| label: clinical_charac
```

Comment:
No harm in using full words, most IDE's, including RStudio automatically recognise when you are typing references and allow you to hit `TAB` to auto-complete, saving you typing.

```suggestion
#| label: clinical_characteristics
```

Same is true for other section `label`.

Abbreviations and acronyms place a burden/over-head on the reader to decipher them, whilst you or I may be familiar with common usages others may not be, so it helps with readability to be explicit.